docs-src/cloud/temporal-cloud-limits-sheet.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: temporal-cloud-limits-sheet
+title: What are the Temporal Cloud default limits?
+sidebar_label: Temporal Cloud default limits
+description: This section describes many of the default settings and limits that apply to Workflow Executions in Temporal Cloud.
+tags:
+  - error
+  - warn
+  - limits
+---
+
+This section describes many of the default settings and limits that apply to Workflow Executions and do not pertain to the entire Temporal Cloud account or individual Namespaces in Temporal Cloud, unless otherwise specified.
```

Comment:
This keeps tripping me up - 

I think we should specify at each default limit whether it applies to a Workflow Execution, a Namespace, or an Account.

```suggestion
This section describes many of the default settings and limits that apply to Workflow Executions, Namespaces, and Accounts while using Temporal Cloud.
```